src/nnue/nnue_feature_transformer.h

```diff
@@ -186,25 +186,28 @@ static constexpr int BestRegisterCount() {
 
     return 1;
 }
-
-static constexpr int NumRegs =
-  BestRegisterCount<vec_t, WeightType, TransformedFeatureDimensions, NumRegistersSIMD>();
-static constexpr int NumPsqtRegs =
-  BestRegisterCount<psqt_vec_t, PSQTWeightType, PSQTBuckets, NumRegistersSIMD>();
     #if defined(__GNUC__)
         #pragma GCC diagnostic pop
     #endif
 #endif
 
 
 // Input feature converter
+template<IndexType TransformedFeatureDimensions>
 class FeatureTransformer {
 
    private:
+    static constexpr bool Small = TransformedFeatureDimensions == TransformedFeatureDimensionsSmall;
```

Comment:
This is really backwards logic.

Reply:
I see your point.  I thought the FeatureTransformers and Networks instantiated at the top of evaluate_nnue.cpp (Lines 43-48) should match so it's clear that they are both created w/ matching dimensions.  No problem if you change it.